src/lib.rs

```diff
@@ -22,7 +22,7 @@
 #![feature(integer_atomics)]
 #![feature(duration_as_u128)]
 #![feature(proc_macro_hygiene)]
-#![feature(const_int_ops)]
+//#![feature(const_int_ops)]
```

Comment:
Please remove it?

Reply:
sorry, I will fix and open another pr. I will close this one first.thx!